src/aims_ui/models/get_endpoints.py

```diff
@@ -0,0 +1,75 @@
+from .endpoint import Endpoint
+from flask import url_for
+
+
+def get_endpoints(called_from = None):
+  # Add new endpoints here for auto-creation on landing page
+  endpoint_data = [
```

Comment:
Constructing the endpoints this way seems a little wasteful, you could simply provide an array of Endpoint objects directly instead of converting them:
`endpoints = [Endpoint(title='Unique ...', url='uprn', desc='Search...'), ...]`

Reply:
Fixed this, it looks far neater!